src/main/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperations.scala

```diff
@@ -471,6 +471,33 @@ object TezosDatabaseOperations extends LazyLogging {
 
   }
 
+  /**
+    * Updates accounts history with information is baker deactivated
+    * @param delegates the full delegates' data
+    * @return a database action that updates accounts_history table and returns the number of updated rows
+    */
+  def updateAccountsHistoryWithIsBakerActive(
```

Comment:
I'm not sure I follow the logic here.

Are we deactivating every delegate account that we store in the first place on the db?

Reply:
no, I've made a simple change right now, but the idea is that when we store accounts we update those which status has changed at given block level with the new status